### ISSUE TITLE
Kite window size is too small to read python docstrings in an efficient manner

### ISSUE THREAD

**Issue by GitHunter0** [kiteco/issue-tracker#613]
Kite is really a great extension, however its window size (specially the width) is too small to properly visually inspect python docstrings in many cases.

Is there a way to customize the size of the window or open in a full page on the browser?

Thank you

**Comment by tracure1337** [comment]
I agree. This is like the thing holding me back from becoming a pro customer. This should really be fixed.

**Comment by tracure1337** [comment]
Also why is this labeled vscode plugin?  This is about the general size of the main window.

**Comment by GitHunter0** [comment]
> This is like the thing holding me back from becoming a pro customer.

For sure, this and the lack of window tabs.
